client/src/components/Nav.tsx

```diff
@@ -416,6 +416,15 @@ const Navigation = ({
         ))}
       </NavDropdown>
 
+      <SidebarLink
+        id="preferences-nav"
+        linkTo="/preferences"
+        handleOnClick={resetPages}
+        setIsMenuLinksOpened={() => setIsMenuLinksOpened(false)}
+      >
+        My Preferences
+      </SidebarLink>
```

Comment:
I now realise my previous comment to "put this under My Work" was ambiguous; I meant: put this _inside_ the My Work sub-menu. But an alternative could be to put it in the user-profile sub-menu on the top right (which currently only has "Sign Out").